File: Python/13772.py
def read(arr):
	x = 0
	l = len(arr)
	for p in range(l):
		if arr[p]=='A'or arr[p]=='D'or arr[p]=='O'or arr[p]=='P'or arr[p]=='Q'or arr[p]=='R':
			x += 1
		elif arr[p] == 'B':
			x += 2
		elif arr[p] == '\0':
			break
	return x

tc = int(input())
for i in range(tc):
	arr = list(input())
	print(read(arr))
	
	

File: C/13772.c
#include <stdio.h>
#define MAX 251
int read(char arr[MAX]);

int main()
{
	int t;
	char arr[MAX];
    getchar();
	scanf("%d", &t);
	for (int i = 0; i < t+1; i++){
		gets(arr);
		printf("%d\n", read(arr));
		for (int y = 0; y < MAX; y++){
			arr[y] = '\0';
		}
	}
	return 0;
}

int read(char arr[MAX])
{
	int x = 0;
	for (int p = 0; p < MAX; p++){
		if (arr[p]=='A'||arr[p]=='D'||arr[p]=='O'||arr[p]=='P'||arr[p]=='Q'||arr[p]=='R'){
			x += 1;
		}
		else if (arr[p] == 'B'){
			x += 2;
		}
		else if (arr[p] == '\0')
			break;
	}

	return x;
}
